examples/client-cardano-transaction/src/main.rs

```diff
@@ -0,0 +1,111 @@
+//! This example shows how to implement a Mithril client and use its features.
+//!
+//! In this example, the client interacts with an aggregator whose URL must be specified in the command to get the data.
+
+use anyhow::anyhow;
+use clap::Parser;
+use slog::info;
+use std::sync::Arc;
+
+use mithril_client::common::TransactionHash;
+use mithril_client::{ClientBuilder, MessageBuilder, MithrilResult, VerifiedCardanoTransactions};
+
+#[derive(Parser, Debug)]
+#[command(version)]
+pub struct Args {
+    /// Genesis verification key.
+    #[clap(long, env = "GENESIS_VERIFICATION_KEY")]
+    genesis_verification_key: String,
+
+    /// Aggregator endpoint URL.
+    #[clap(long, env = "AGGREGATOR_ENDPOINT")]
+    aggregator_endpoint: String,
```

Comment:
Default values would be welcome for theses parameters.
Same comment for other crates.